glib/src/param_spec.rs

```diff
@@ -876,6 +999,13 @@ impl ParamSpecFlags {
     }
 }
 
+define_builder!(
+    ParamSpecFlags,
+    ParamSpecFlagsBuilder,
+    flags_type: crate::Type = crate::Type::FLAGS,
```

Comment:
The flags type must be a concrete flags type and is mandatory, so must be given in the `new()` / `builder()` function